tests/testthat/test-make-style.r

```diff
@@ -30,13 +30,13 @@ test_that("hexa color regex works", {
                 "foo#123456bar")
 
   for (color in positive) {
-    expect_true(grepl(hash_color_regex, color))
-    expect_true(grepl(hash_color_regex, toupper(color)))
+    expect_true(grepl(crayon:::hash_color_regex, color))
+    expect_true(grepl(crayon:::hash_color_regex, toupper(color)))
```

Comment:
Actually, this is also the same thing that tripped me up with `%:%`, not to mention I apparently don't know how to grep on the command line.